src/Runner.Worker/ExecutionContext.cs

```diff
@@ -749,6 +749,10 @@ public void InitializeJob(Pipelines.AgentJobRequestMessage message, Cancellation
             {
                 githubContext["job"] = new StringContextData(githubJob);
             }
+            if (!string.IsNullOrEmpty(message.JobDisplayName))
+            {
+                githubContext["job_display_name"] = new StringContextData(message.JobDisplayName);
+            }
```

Comment:
I did try to write a test for this, but was having a hard time finding the minimal amount of setup required to initialize an execution context and populate this. Before investing more time, I figured I'd open this and see if there was interest in the addition.

Reply:
Thanks @kylekthompson , there's a relatively quick internal process for adding new context vars other than the runner change (update docs, auditing, consider future supportability etc). I'll get the ball rolling and will get back to you. 

Once it's green lit, as you suggested,
 we can look at some unit-tests for it.

Reply:
I'd also recommend taking the feedback you had regarding the API and opening a discussion about it on the [community](https://github.com/orgs/community/discussions/) forum / discussions to get some eyes on it :)

Reply:
Much appreciated! I'll go ahead and open a discussion there as well, thanks!

Reply:
For posterity, here’s the community discussion post: https://github.com/orgs/community/discussions/38277